lib/util/cache/repository/index.spec.ts

```diff
@@ -36,16 +36,15 @@ describe('util/cache/repository/index', () => {
     expect(isCacheModified()).toBeUndefined();
   });
 
-  it('skips saves cache on dry run', async () => {
+  it('saves cache on dry run', async () => {
```

Comment:
I don't think this is a good idea, if you share the cache with normal renovate runs, the full run after the dry run would do nothing because the cache is up-to-date.

Reply:
Good point. Extract cache could be saved though

Reply:
@rarkins @viceice 
The extract cache is already being used regardless.
ill remove saving cache,